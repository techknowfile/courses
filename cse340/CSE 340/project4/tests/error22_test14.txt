VAR
    x, y : STRING;
    z : x;
{
    y = x;
}
