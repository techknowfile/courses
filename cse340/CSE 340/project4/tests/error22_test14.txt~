VAR
    x, y : STRING;
    z : x;
{
}
