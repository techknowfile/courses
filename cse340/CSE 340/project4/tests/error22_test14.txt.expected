ERROR CODE 2.2 x
